addons/mail/static/src/new/discuss/discuss.xml

```diff
@@ -29,7 +29,7 @@
                 <div class="o-mail-discuss-actions flex-shrink-0 d-flex align-items-center ms-1">
                     <button t-if="thread.id === 'inbox'" class="btn btn-secondary" t-att-disabled="thread.isEmpty" t-on-click="markAllAsRead">Mark all read</button>
                     <button t-if="thread.id === 'starred'" class="btn btn-secondary" t-att-disabled="thread.isEmpty" t-on-click="unstarAll">Unstar all</button>
-                    <button t-if="thread.type !== 'mailbox' and thread !== rtc.state.channel" class="btn" title="Start a Call" t-on-click="() => this.rtc.toggleCall(this.thread)">
+                    <button t-if="thread.allowCalls and thread.type !== 'mailbox' and thread !== rtc.state.channel" class="btn" title="Start a Call" t-on-click="() => this.rtc.toggleCall(this.thread)">
```

Comment:
`and thread.type !== 'mailbox'` looks overkill with `allowCalls` (can be cleaned in a follow up PR)